core/src/avm2/globals/flash/ui/GameInputDevice.as

```diff
@@ -1,4 +1,9 @@
 package flash.ui {
     public final class GameInputDevice {
+        // Specifies the maximum size for the buffer used to cache sampled
+        // control values. If `startCachingSamples` returns samples that
+        // require more memory than you specify, it throws a memory error.
+        [API("689")]
```

Comment:
This entire class should be `[API("688")]`

Reply:
Right - but not 689? It looks AIR-only to me.

Reply:
I can access it in normal FP

Reply:
![image](https://github.com/ruffle-rs/ruffle/assets/29206584/426be721-b0bd-48cf-b386-e68d9e878fdc)

Reply:
Thanks, fixed!